Library/Homebrew/os/linux.rb

```diff
@@ -31,12 +31,6 @@ def os_version
   module Mac
     module_function
 
-    # rubocop:disable Naming/ConstantName
-    # rubocop:disable Style/MutableConstant
-    ::MacOS = OS::Mac
```

Comment:
Ideally/eventually: this won't be defined on Linux at all.

Reply:
Currently, we need the `MacOS::Versions::SYMBOLS` constant to be defined on Linux, too. This is because this constant is used to generate the `on_{macos_version}` blocks and we need those to be defined on Linux, too. Is there a better place for those versions to be defined?

It's also worth noting that even on Linux, it seems like macOS version of `OS::Mac` is still loaded because there are a few places where cask files are required even on Linux. This should probably be removed but is outside the scope of this PR.

Reply:
> Is there a better place for those versions to be defined?

A dedicated `Library/Homebrew/macos_versions.rb` might make sense? Shouldn't block this PR.

> This should probably be removed but is outside the scope of this PR.

Agreed 👍🏻